Simbody/include/simbody/internal/SmoothSphereHalfplaneContact.h

```diff
@@ -143,6 +142,9 @@ class SimTK_SIMBODY_EXPORT SmoothSphereHalfplaneContact : public Force {
      * @param dynamicFriction       the coefficient of dynamic friction (ud)
      * @param viscousFriction       the coefficient of viscous friction (uv)
      * @param transitionVelocity    the transition velocity (vt)
+     * The default values are 1 for stiffness, 0 for dissipation, 
```

Comment:
I wonder whether this where we should document the default values. This is part of the setParameters method whereas Sherm suggests documenting what the default values are if someone doesn't call any of the set methods. Do you see another place where this would fit? We should perhaps look into other methods for examples.